Build/net45.props

```diff
@@ -72,7 +72,7 @@
     <Features>$(Features);FEATURE_ASYNC</Features>
     <Features>$(Features);FEATURE_GET_TYPE_INFO</Features>
     <Features>$(Features);FEATURE_DYNAMIC_EXPRESSION_VISITOR</Features>
-    <Features>$(Features);FEATURE_ASSEMBLYBUILDER_DEFINEDYNAMICASSEMBLY</Features>
+    <Features Condition=" '$(Mono)' != 'True' ">$(Features);FEATURE_ASSEMBLYBUILDER_DEFINEDYNAMICASSEMBLY</Features>
```

Comment:
It's even more confusing than that because Build.proj sets a property as well. It is lowercase though